tests/backup/backup_resiliency_test.go

```diff
@@ -1019,3 +1021,166 @@ var _ = Describe("{ScaleMongoDBWhileBackupAndRestore}", func() {
+				backupNames = append(backupNames, fullBackupName)
+				err = CreateBackup(fullBackupName, SourceClusterName, customBackupLocationName, backupLocationUID, []string{namespace},
+					labelSelectors, orgID, clusterUid, "", "", "", "", ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying backup [%s] creation", fullBackupName))
+			}
+
+			// Incremental backup
+			for _, namespace := range bkpNamespaces {
+				incrementalBackupName = fmt.Sprintf("%s-%s-%v", "incremental-backup", namespace, time.Now().Unix())
+				incrementalBackupNames = append(incrementalBackupNames, incrementalBackupName)
+				err = CreateBackup(incrementalBackupName, SourceClusterName, customBackupLocationName, backupLocationUID, []string{namespace},
+					labelSelectors, orgID, clusterUid, "", "", "", "", ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying backup [%s] creation", incrementalBackupName))
+			}
+			log.Infof("List of backups - %v", backupNames)
+			log.Infof("List of Incremental backups - %v", incrementalBackupNames)
+
+		})
+		Step("Deleting incremental backup", func() {
+			log.InfoD("Deleting incremental backups")
+			backupDriver := Inst().Backup
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, backupName := range incrementalBackupNames {
+				log.Infof("About to delete backup - %s", backupName)
+				backupUID, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
+				log.FailOnError(err, "Failed while trying to get backup UID for - %s", backupName)
+				_, err = DeleteBackup(backupName, backupUID, orgID, ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Deleting backup - [%s]", backupName))
+			}
+		})
+		Step("Taking incremental backups of applications again", func() {
+			log.InfoD("Taking incremental backups of applications again")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			// Incremental backup
+			for _, namespace := range bkpNamespaces {
+				incrementalBackupName = fmt.Sprintf("%s-%s-%v", "incremental-backup", namespace, time.Now().Unix())
+				incrementalBackupNames = append(incrementalBackupNames, incrementalBackupName)
+				incrementalBackupNamesMap[incrementalBackupName] = incrementalBackupName
+				err = CreateBackup(incrementalBackupName, SourceClusterName, customBackupLocationName, backupLocationUID, []string{namespace},
+					labelSelectors, orgID, clusterUid, "", "", "", "", ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying backup [%s] creation", incrementalBackupName))
+			}
+			log.Infof("List of New Incremental backups - %v", incrementalBackupNames)
+		})
+		Step("Check if backups are incremental backups or not", func() {
+			log.InfoD("Check if backups are incremental backups or not")
+			var bkp *api.BackupObject
+			backupDriver := Inst().Backup
+			bkpEnumerateReq := &api.BackupEnumerateRequest{
```

Comment:
Instead of doing enumerate we can just do inspect backup. Since we already know the name of the backup which we will be validating all we need to do is inspect and get the response and use that response to get the `BackupInfo_Volume`

Reply:
We can eliminate nested for loop here

Reply:
Done